### PR TITLE
Add distro options to host.os.py's command line parser

### PR DESCRIPTION
**Pull request by alanoe**

### CODE REVIEW

File: lib/config.py [hunk 1]
```diff
@@ -50,6 +50,12 @@
         dict(help='Keep build directory and its logs and artifacts.',
              action='store_true'),
 }
+DISTRO_ARGS = {
+    ('--distro-name',):
+        dict(help='Base Linux distribution', default=''),
```
Comment (ghost) [inline lib/config.py]: Why have this default? The default value `None` is just as good, and it will be replaced by the value in `config.yaml` anyway.

Comment (alanoe) [reply lib/config.py]: Done

**Review by ghost — APPROVED** [review]

**Review by farosas — APPROVED** [review]

**Comment by ghost** [comment]
@alanoe Mentioned that we're probably not gonna need https://github.com/open-power-host-os/builds/pull/180/commits/91e6affefa1c89a1daea92defee20c76f4b5a069. Let's put this PR on hold until we have the Jenkins jobs for validating previous versions and make sure they work with this code.

**Comment by alanoe** [comment]
start build

**Pull request merged**